packages/app/src/elements/settings-security.ts

```diff
@@ -783,7 +811,8 @@ export class SettingsSecurity extends StateMixin(Routing(LitElement)) {
                         </div>
 
                         ${this._renderBiometricUnlock()} ${this._renderMFA()} ${this._renderSessions()}
-                        ${this._renderTrustedDevices()} ${this._rendersecurityReport()}
+                        ${this._renderTrustedDevices()} ${this._renderSecurityReport()}
+                        ${this._renderEmailNotifications()}
```

Comment:
While this is an "account setting" (not "app setting"), I think it still belongs in this settings -> security screen.